ocaml/runtime/fiber.c

```diff
@@ -111,7 +114,34 @@ Caml_inline struct stack_info* alloc_for_stack (mlsize_t wosize)
   si->size = len;
   return si;
 #else
+#ifdef NATIVE_CODE
+  size_t bsize = sizeof(value) * wosize;
```

Comment:
I think the canonical way of doing this in the runtime is `Bsize_wsize(wosize)`.